src/Components/CreateAccountForm/CreateAccountForm.tsx

```diff
@@ -1,11 +1,196 @@
+import "./CreateAccountForm.css"
 import React from 'react'
+import { useState } from 'react'
 
 function CreateAccountForm() {
+  const [formData, setFormData] = useState({
+    about: "",
+    address: {
+      street: "",
+      city: "",
+      state: "",
+      zipcode: "",
+    },
+    email: "",
+    first_name: "",
+    last_name: "",
+    isRemote: false ,
+    id: "",
+    type: "",
+    skills: [],
+  })
+
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    const { name, value } = e.target;
+  
+    if (name.includes("address.")) {
```

Comment:
took me a min to see what this was doing, but very cool!!! great way to DRY up your code